connector/mqtt/vernemq/broker/src/dojot_disconnect_plugin/src/clean_sess.erl

```diff
@@ -1,13 +1,14 @@
 -module(clean_sess).
```

Comment:
It doesn't seem a good name for this module. What about `helpers.erl` or `utils.erl`?